services/QuillLMS/app/models/question_health_dashboard.rb

```diff
@@ -25,7 +25,11 @@ def percent_reached_optimal_for_question
   end
 
   def cms_dashboard_stats
-    JSON.parse(cms_data.body)
+    begin
+      JSON.parse(cms_data.body)
+    rescue JSON::ParserError
+      {}
+    end
```

Comment:
Cool syntax trick that you can use here if you want: for very simple functions like this one, you can `rescue` the whole thing without a `begin` block:
```ruby
def cms_dashboard_stats
  JSON.parse(cms_data.body)
rescue JSON::ParserError
  {}
end
```